### DISCUSSION TITLE
Agregar un campo notas en la info de la task

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#92] [3. Ideas & Feature Requests] [answered]
Hola

Me gustaría que si pueden considerar lo siguiente:

Agregar un campo de Notas Adicionales en la vista de una task, y cuya nota, la puede agregar todos los involucrados de la tasks, indicando la fecha de ingreso + quien la registró + el comentario.

Algo asi

![image](https://user-images.githubusercontent.com/61419471/115247561-e175f300-a0f4-11eb-8b19-4a53b4424fbb.png)

¿Cuál es la diferencia entre Info y este campo?

**Info**: es un solo campo que se edita entre sí, perdiendo alguna información anterior (a excepción que se guarde en alguno lado en la librería de la tasks

**Campo Notas**: Puedes agregar muchas notas sin borrar la información anterior.

En qué caso se podría realizar

1.- Si tengo una oportunidad de un cliente y mi tarea es seguimiento a cliente X. se podría agregar algo asi

1.- 18-04-2021, no me pude contactar con el cliente
2.- 19-04-2021 agendé una reunión con el cliente
3.- 20-04-2021, me piden una propuesta formal
Etc.

Gracias

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG esto también lo puedes hacer con el campo info, puedes concatenar información anterior, con nueva información, ej:

`$JOIN#```\n```#($VALUE#task|info)#"Información adicional"`

**Reply by NicolasPLG** [reply]
@guillevalin pero el $JOIN# se utilizaría en un survey trigger y lo que estoy indicando es desde la vista task. En mi caso en algunas implementaciones dejo el info como el id de una 2° task para hacer referencia entre máquinas (puede que sea otra pedida aparte, el de agregar un campo no editable para colocar un id de una tasks.

**Reply by guillevalin** [reply]
@NicolasPLG 

No es buena práctica hacer eso dado que el campo info es editable por el usuario en la misma vista de tareas. Y no veo problemas con usar $JOIN dentro de un survey trigger, siempre puedes acceder a la información de la tarea (o pedirla con NWRequest). Mi recomendación respecto al campo para guardar tareas es que puedas tener una de las máquinas con asset "unique", por lo que se crearia una property para cada "task", y en el propertyType del asset, definir un schema donde puedas guardar ID de tareas, que sería una forma más correcta de hacerlo.

**Reply by acedward** [reply]
1. Como nota adicional se vienen tipo de datos extras para Tasks. Podrás agregar campos dinámicos.
2. En general tu pregunta se puede añadir datos vía formularios (crear un formulario, y enviarlo en el canal de la tarea automáticamente se agregará la información al canal)

**Reply by NicolasPLG** [reply]
Los extras de las tasks estaría muy bueno. Mi pregunta no era directa por un formulario, sino en la misma vista (como cambiar el estado actual de la task, las fechas, etc, agregarlo desde allí).
Gracias por las respuesta
